velox/functions/lib/string/StringImpl.h

```diff
@@ -173,6 +173,64 @@ std::vector<int32_t> stringToCodePoints(const T& inputString) {
   return codePoints;
 }
 
+template <typename TInString>
+FOLLY_ALWAYS_INLINE int64_t
+hammingDistanceUnicode(const TInString& left, const TInString& right) {
+  int64_t distance = 0;
+  int64_t leftPosition = 0;
+  int64_t rightPosition = 0;
+  std::string_view leftSV = std::string_view(left.data(), left.size());
+  std::string_view rightSV = std::string_view(right.data(), right.size());
+  int64_t leftLength = leftSV.size();
+  int64_t rightLength = rightSV.size();
+
+  while (leftPosition < leftLength && rightPosition < rightLength) {
+    int leftSize, rightSize;
```

Comment:
These needn't be defined within the while loop. They can be initialized outside it.

Reply:
Sure. Moved it outside loop.